Source/WebCore/animation/StyleOriginatedTimelinesController.cpp

```diff
@@ -126,13 +126,24 @@ ScrollTimeline* StyleOriginatedTimelinesController::determineTreeOrder(const Vec
         // Has blocking timeline scope element
         if (containsElement(timelineScopeElements, element.get()))
             return nullptr;
-        element = element->parentElement();
+        element = element->parentElementInComposedTree();
     }
 
     ASSERT_NOT_REACHED();
     return nullptr;
 }
 
+static bool elementIsDescendantOfOrOtherElement(const Element& child, const Element& potentialParent)
```

Comment:
How about `elementIsComposedDescendantOf` as name?

Reply:
Antti also pointed out `elementIsParentOfElementInComposedTree` as something to potentially reconcile with.

Reply:
I've introduced a new `bool Node::isParentOfNodeInComposedTree(const Node&) const` method to replace a static function previously introduced in [291349@main](https://commits.webkit.org/291349@main).